### ISSUE TITLE
Feature Request: Consistent Positional Buttons for Viper

### ISSUE THREAD

**Issue by Lunatory** [Grammernatzi/XIVComboPlugin#193]
This is kind of a tougher one and might simplify things too much for what you're going for with this tweak, but I would really like an option to have my enhanced Hindsbane Fang / Hindsting Strike go over my Reaving Fangs combo (leaving the Steel Fangs combo untouched) and vice-versa for Flanksting Strike and Flanksbane Fang over Steel Fangs. I drew a poorly illustrated image to explain my request better.
![feature req](https://github.com/user-attachments/assets/d3f0f2d5-a363-429e-8e62-287590ebdfd9)

In other words, when the player has the Hindstung Venom buff, can you replace Hindsbane Fang with Hindsting Strike, and when the player has the Flanksbane Venom buff, can you change Flanksting Strike to Flanksbane Fang?

**Comment by Grammernatzi** [comment]
Since this leans into the more 'lazy' style of things rather than saving buttons, I'll probably defer to saying you should ask MKhayle on regular Expanded to see if it is alright with them, too. They did something similar with Steel/Reaving Fangs, so I'll add that for consistency's sake, at least.

**Comment by Lunatory** [comment]
Thank you! I posted a feature request on their xivcombo page.

**Comment by Grammernatzi** [comment]
I see it was accepted, so I will go ahead and add it, then. I just wanted to make sure; thank you for your patience!

**Comment by Lunatory** [comment]
Though unlikely to occur the current implementation can lead to completely locking the user out of actions, if you mis-press the unenhanced 2nd combo you are offered an un-executable action over both combos. Since the logic for the 2nd action seems to be tracked using some hidden data (not quite buff related since using Vicewinder to alter what second stage button lights up), the best course of action would likely be to check if the previously used action was Swiftskin's Sting before rear positionals, or Hunters' String before side positionals.

**Comment by Lunatory** [comment]
Alternatively both second stage buttons could be changed to be the enhanced one as well to prevent mis-presses altogether.

**Comment by Grammernatzi** [comment]
My bad on the mispress thing, I somehow did not notice that. This has been hotfixed. And I'm not touching the second stage thing, it's already basic enough as it is with both of these new things.
